doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -10,6 +10,6 @@
         "#1": "Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For a sqlite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
         "#3": "Set \"serverVersion\" to your server version to avoid edge-case exceptions and extra database calls",
-        "DATABASE_URL": "mysql://root@127.0.0.1:3306/symfony?charset=utf8mb4&serverVersion=5.7"
+        "DATABASE_URL": "mysql://app:app@127.0.0.1:3306/app?charset=utf8mb4&serverVersion=5.7"
```

Comment:
Can I propose that we rename `DATABASE_URL` to `DATABASE_DSN`?

http://php.net/manual/en/pdo.construct.php
http://php.net/manual/en/ref.pdo-mysql.connection.php
http://php.net/manual/en/ref.pdo-pgsql.connection.php
http://php.net/manual/en/ref.pdo-sqlite.connection.php